hera_pspec/pspecdata.py

```diff
@@ -3756,7 +3851,7 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
                             pv[:,t] = pv[:,t] * sd
 
                     #Generate the covariance matrix if error bars provided
-                    if store_cov:
+                    if store_cov or store_QE_error:
```

Comment:
Is this needed? if store_QE_error is True, then certainly store_cov is true, right?

Reply:
store_cov and store_cov_diag would be independent choices here, while the former one determines if we store the whole cov_array, and the latter one determines if we store only its diagonal.

Reply:
right, but I think we should do one OR the other, not both. That would be storing redundant information